source/admin_manual/list_of_events.rst

```diff
@@ -164,646 +164,289 @@ SQL Categories
-+=====================+======================================================+
-| mode                | Values:                                              |
-|                     |  * ``allow``                                         |
-|                     |  * ``report``                                        |
-+---------------------+------------------------------------------------------+
-| policy_response     | Value returned from policy server (number)           |
-|                     | Otherwise, the driver name.                          |
-+---------------------+------------------------------------------------------+
+   :field user: Full username. This can change during authentication, for
+     example due to passdb lookups.
+   :field original_user: Original username exactly as provided by the client.
+   :field translated_user: Similar to ``original_user``, except after
+     :dovecot_core:ref:`auth_username_translation` translations are applied.
+   :field login_user: When doing a master user login, the user we are logging
+     in as. Otherwise not set.
+   :field master_user: When doing a master user login, the master username.
+     Otherwise not set.
+   :field mechanism @added;v2.3.12: Name of used SASL mechanism (e.g. PLAIN).
+   :field service @added;v2.3.12: Service doing the lookup (e.g. ``imap``,
+     ``pop3``, ...).
+   :field session @added;v2.3.12: Session ID.
+   :field client_id @added;v2.3.12: Expands to client ID request as IMAP
+     arglist. Needs ``imap_id_retain=yes``.
+   :field remote_ip @added;v2.3.12: Remote IP address of the client connection.
+   :field local_ip @added;v2.3.12: Local IP address where client connected to.
+   :field remote_port @added;v2.3.12: Remote port of the client connection.
+   :field local_port @added;v2.3.12: Local port where the client connected to.
+   :field real_remote_ip @added;v2.3.12: Same as ``remote_ip``, except if the
+     connection was proxied this is the proxy's IP address.
+   :field real_local_ip @added;v2.3.12: Same as ``local_ip``, except if the
+     connection was proxied this is the proxy's IP where proxy connected to.
+   :field real_remote_port @added;v2.3.12: Same as ``remote_port``, except if
+     the connection was proxied this is the proxy connection's port.
+   :field real_local_port @added;v2.3.12: Same as ``local_port``, except if
+     the connection was proxied this is the local port where the proxy
+     connected to.
+   :field local_name @added;v2.3.12: TLS SNI hostname, if given.
+   :field transport @added;v2.3.12:
+     Client connection's transport security. Values:
+       * ``insecure``
+       * ``trusted``
+       * ``TLS``
 
+.. dovecot_event:field_group:: auth_server_passdb
 
-Authentication Client
-=====================
+   :field passdb: Driver name.
+   :field passdb_name: ``passdb { name }``, if it is configured. Otherwise,
+     same as ``passdb``.
```

Comment:
This sounds confusing here. The original text was "Otherwise, the driver name." meaning passdb { driver }. So maybe here also say "same as passdb { driver }"?